src/components/Category.js

```diff
@@ -0,0 +1,171 @@
+import React from "react";
+import styled from "styled-components/macro";
+import Title from "./common/Title";
+
+// import sdf from "../../public/"
+
+export default function Category({ firstPart, secondPart }) {
+  function CategoryGrid() {
+    return (
+      <GridContainerStyed>
+        <CategoryStyled>
+          <a href="/business">
```

Comment:
You could use the React Router `<Link>` here instead of `<a>`, so it's clearer that you use the Router here: https://reacttraining.com/react-router/web/guides/quick-start/1st-example-basic-routing